### PR TITLE
move shib require to top location block

### PR DESCRIPTION
**Pull request by umjnega**
Went ahead and looked into the fulcrum issue from yesterday and I think I correclty updated the change @skorner made.

### CODE REVIEW

File: manifests/profile/www_lib/vhosts/fulcrum.pp [hunk 1]
```diff
@@ -154,12 +154,6 @@
       XSendFile on
       XSendFilePath ${derivatives_path}
       XSendFilePath ${alt_derivatives_path}
-      # Configure Shibboleth for authentication via InCommon partner login
-      <Location "/">
-        AuthType shibboleth
-        ShibRequestSetting requireSession 0
```
Comment (skorner) [inline manifests/profile/www_lib/vhosts/fulcrum.pp:160]: These two lines need to be present in the rendered `Location /` definition (see the diff I posted in slack). You can do this with an auth_require and custom fragment line, as is done [here](https://github.com/mlibrary/nebula/blob/cdc18854b8ab966eeed0f4b360b58474d86be032/manifests/profile/www_lib/vhosts/staff_lib.pp#L56-L60) (ignore the auth_require in that example as that is already covered with the require all definition in this manifest). You can probably make this addition to authz_base_requires.

**Review by skorner — CHANGES_REQUESTED** [review]
See the comment I added that highlights one more change we have to address for this to work.

File: manifests/profile/www_lib/vhosts/fulcrum.pp (hunk 1 above)
Comment (umjnega) [reply manifests/profile/www_lib/vhosts/fulcrum.pp:160]: auth_require or auth_type...  I may have pushed my last commit the opposite.  But i'll let you correct me if i'm reading this wrong.

**Review by skorner — APPROVED** [review]

File: manifests/profile/www_lib/vhosts/fulcrum.pp (hunk 1 above)
Comment (skorner) [reply manifests/profile/www_lib/vhosts/fulcrum.pp:160]: Looks good to me! Go ahead and squash to 1 commit and merge after the tests pass from the squash.

**Pull request merged**